frontend/src/pages/org/archived-item-qa/ui/text.ts

```diff
@@ -29,33 +29,47 @@ function renderDiff(
           aria-labelledby="crawlTextHeading"
         >
           ${diff.map((part) => {
-            return html`
-              <span
-                class=${part.added
-                  ? removedText
-                  : part.removed
-                    ? addedText
-                    : ""}
+            if (part.added) {
+              return html`<del
+                aria-label="${msg("Missing text on crawl")}"
+                class="${removedText}"
+                >${part.value}</del
+              >`;
+            } else if (part.removed) {
+              return html`<ins
+                aria-label="${msg("Added text on crawl")}"
+                class="${addedText}"
+                >${part.value}</ins
+              >`;
+            } else {
+              return html`<span aria-label="${msg("Identical text")}"
                 >${part.value}</span
-              >
-            `;
+              >`;
+            }
           })}
         </div>
         <div
           class=${tw`flex-1 overflow-hidden border-dashed p-4 first-of-type:border-r`}
           aria-labelledby="qaTextHeading"
         >
           ${diff.map((part) => {
-            return html`
-              <span
-                class=${part.added
-                  ? addedText
-                  : part.removed
-                    ? removedText
-                    : ""}
+            if (part.added) {
+              return html`<ins
+                aria-label="${msg("Added text on analysis")}"
+                class="${addedText}"
+                >${part.value}</ins
+              >`;
+            } else if (part.removed) {
+              return html`<del
+                aria-label="${msg("Missing text on analysis")}"
+                class="${removedText}"
+                >${part.value}</del
+              >`;
+            } else {
+              return html`<span aria-label="${msg("Identical text")}"
```

Comment:
Is the label here necessary? Seems it could get very verbose, and the insertion/deletion labels likely would give enough context to know everything else has stayed the same?

Reply:
The label here is essentially required.  `<ins>` and `<del>` aren't reliably read aloud by all screen readers.